app/commands/donations/stripe/sync_subscriptions.rb

```diff
@@ -0,0 +1,48 @@
+# Ensure that our local Stripe subscriptions match the actual Stripe subscriptions
+class Donations::Stripe::SyncSubscriptions
+  include Mandate
+
+  def call
+    create_subscriptions!
+    deactivate_subscriptions!
+  end
+
+  def create_subscriptions!
+    missing_active_subscription_ids = active_stripe_subscription_ids - local_subscription_ids
+    missing_active_subscription_ids.each do |subscription_id|
+      subscription = active_stripe_subscriptions[subscription_id]
+
+      user = User.find_by(stripe_customer_id: subscription.customer)
+      next unless user
```

Comment:
If this is missing, I wonder if we should set the `stripe_customer_id` by syncing on the email field coming back from Stripe?

Reply:
Yeah we can do that